web/base_viewer.js

```diff
@@ -1476,6 +1478,35 @@ class BaseViewer {
     this._setCurrentPageNumber(pageNumber, /* resetCurrentPageView = */ true);
     this.update();
   }
+
+  /**
+   * @type {number} One of the values in {ReadingDirection}.
+   */
+  get direction() {
+    return this._direction;
+  }
```

Comment:
```suggestion
  get readingDirection() {
    return this._readingDirection;
  }
```
(And similar in the getter below.)